services/ui-src/src/components/FormPage/FormPage.scss

```diff
@@ -11,3 +11,17 @@
 .action-button {
   margin: 0.6rem 0rem 2rem 5rem;
 }
+
+.formfooter{
+  .grid-container{
+    max-width: none;;
```

Comment:
Request: Would you mind removing the double `;;` here, and adding a newline at the end of the file?